tests/Requests/AbstractCommandTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace OpenEuropa\EPoetry\Tests\Requests;
+
+use donatj\MockWebServer\MockWebServer;
+use OpenEuropa\EPoetry\Tests\AbstractTest;
+
+/**
+ * Class AbstractRequestTest.
```

Comment:
Add meaningful class description.

Reply:
For consistency, I will keep this class description.







:)

I'm kidding, it's good to start changing and having good practices, even if it's not consistent with other classes.